### PR TITLE
Undeploy of solution.

### PR DESCRIPTION
**Pull request by neza-vehovar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,3 +12,8 @@ prerequisites:
 * find opera orchestrator here: https://github.com/xlab-si/xopera-opera, install it and run this blueprint with
 
 `opera deploy <some deploy name> resize_service_opera.yml`
+
+
+undeploy solution with:
+
+`opera deploy <some undeploy name> resize_service_opera_undeploy.yml`
```
Comment (cankarm) [inline README.md]: I think that we are missing the point here. The `opera` has already the `undeploy` action, see:
```
(.venv) matic@MC-COMP:~/Documents/XLAB/Projects/RADON/03_git/xOpera/opera$ opera
error: the following arguments are required: command
usage: opera [-h] {deploy,undeploy} ...

opera orchestrator

positional arguments:
  {deploy,undeploy}
    deploy           Deploy service template from CSAR
    undeploy         Undeploy service template

optional arguments:
  -h, --help         show this help message and exit
```

Therefore you do not need to make new yaml, Only populate current one with instructions for undeploy/delete. See [here](https://github.com/xlab-si/xopera-opera/blob/master/examples/service.yaml) the `delete` parts.

File: playbooks/aws_role/undeploy.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+- hosts: all
```
Comment (cankarm) [inline playbooks/aws_role/undeploy.yml]: Be careful with this whole undeploy block. Can you check with ansible if there is another function/action related with this role?

File: playbooks/lambda/create.yml [hunk 3]
```diff
@@ -10,6 +10,9 @@
         role: "{{ aws_role }}"
         handler: image_resize.lambda_handler
       register: lambda_info
+      # If a new aws_role is created there is 5-6sec delay before lambda can assume the role
+      until: lambda_info is succeeded
+      delay: 6
```
Comment (cankarm) [inline playbooks/lambda/create.yml]: This is not relevant only for `undeploy` therefore it should be added as a separate pull request.

File: resize_service_opera_undeploy.yml [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        - host: my-workstation
+
+    aws_lambda_role:
+      type: radon.policies.createRole
+      properties:
+        aws_region: eu-central-1
+        role_name: RadonLambda
+        role_description: Lambda role
+      requirements:
+        - host: my-workstation
```
Comment (cankarm) [inline resize_service_opera_undeploy.yml]: Remove this file. Not needed.

**Review by cankarm — CHANGES_REQUESTED** [review]
The `undeploy` is not correctly implemented - see other comments.

**Comment by neza-vehovar** [comment]
Changes were done.

**Review by cankarm — APPROVED** [review]

**Comment by cankarm** [comment]
I tested with xOpera and it successfully removes function and buckets!
LGTM!

**Pull request merged**